index.html

```diff
@@ -1,657 +1,1338 @@
+                      </div>
+                    </div>
+
+                    <div class="container responsivegrid hamburger">
+                      <div id="container-686bb06b29" class="cmp-container">
+                        <div class="image open">
+                          <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/container/image_324769233.coreimg{.width}.svg/1669059161137/hamburger.svg" data-asset="/content/dam/LeaderImpact/site-./assets/hamburger.svg" data-title="menu" id="image-479635272c" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                            <img src="./assets/hamburger.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="menu">
+                          </div>
+                        </div>
+
+                        <div class="image close">
+                          <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/container/image.coreimg{.width}.svg/1669059409586/close.svg" data-asset="/content/dam/LeaderImpact/site-./assets/close.svg" data-title="Close" id="image-534f826118" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                            <img src="./assets/close-menu.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
                 </div>
+              </div>
             </div>
-            <div class="mobile-nav-wrap">
-                <nav id="" class="nav-wrap" itemscope="" itemtype="" data-cmp-data-layer="">
-                    <ul class="nav-link-wrap">
-                        <li class="navigation-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Join a Group</a>
-                        </li>
-                        <li class="navigation-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Start a Group</a>
-                        </li>
-                        <li class="navigation-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Resources</a>
-                        </li>
-                        <li class="navigation-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">About</a>
-                        </li>
+
+            <div class="container responsivegrid mobile-navigation">
+              <div id="container-4980578f66" class="cmp-container">
+                <div class=" mobile-nav-wrap">

```

Comment:
This also has class `navigation` on it, which unfortunately makes it also a `display: none;` on mobile. We'll need to add a responsive style to show `.mobile-nav-wrap` which overrides the `display: none;` on the `.navigation` class.

Reply:
Can you elaborate this one a bit more please?